### PR TITLE
landingpage updates

### PR DESCRIPTION
**Pull request by jetzlstorfer**

### CODE REVIEW

File: content/home/kubernetes-cluster-step2.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-And deploy Keptn on your cluster
+And deploy Keptn on your Kubernetes cluster
 
 ```console
-keptn install --platform=[aks|eks|gke|openshift|pks|kubernetes]
+keptn install --use-case=[continuous-delivery,quality-gates]
```
Comment (johannes-b) [inline content/home/kubernetes-cluster-step2.md:4]: The `quality-gates` flag is gone. 

Please change to: `keptn install (--use-case=continuous-delivery)`

File: content/home/kubernetes-cluster-step2.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 ```
 
 \
-Keptn will be installed with the control plane and all batteries-included services.
+Keptn will be installed with all batteries-included services or with the control-plane and quality-gates functionality only.
```
Comment (johannes-b) [inline content/home/kubernetes-cluster-step2.md:8]: Please change to: 

Keptn will be installed with its control-plane for quality gates and automated operations. If the `--use-case` flag is set, the services for continuous delivery will be installed too.

File: content/home/automatic-remediation-right.md [hunk 3]
```diff
@@ -1,12 +1,21 @@
...
+apiVersion: spec.keptn.sh/0.1.4
+kind: Remediation
+metadata:
+  name: serviceXYZ-remediation
+spec:
+  remediations:  
+  - problemType: Response time degradation
+    actionsOnOpen:
+    - name: Scaling ReplicaSet by 1
+      description: Scaling the ReplicaSet of a Kubernetes Deployment by 1
```
Comment (johannes-b) [inline content/home/automatic-remediation-right.md:11]: Please shorten to: `Scales the ReplicaSet of a Deployment`

(to remove the horizontal scrolling bar)

File: content/home/automatic-remediation-right.md [hunk 4]
```diff
@@ -1,12 +1,21 @@
...
+      action: scaling
+      value: "1"
+    - name: Toogle feature flag
+      action: featuretoggle
+      description: Toggle feature flag EnablePromotion from ON to OFF.
```
Comment (johannes-b) [inline content/home/automatic-remediation-right.md:16]: Please shorten to: `Toggles feature flag EnablePromotion`

(to remove the horizontal scrolling bar)

File: content/home/kubernetes-cluster-step2.md (hunk 1 above)
Comment (jetzlstorfer) [reply content/home/kubernetes-cluster-step2.md:4]: Not so according to the docs: https://keptn.sh/docs/0.7.x/reference/cli/commands/keptn_install/ 
Is it completely gone or just the default?

Comment (johannes-b) [reply content/home/kubernetes-cluster-step2.md:4]: It is the default one. 

Unfortunately, the documentation for the `keptn install` command has not been updated. (FYI: https://github.com/keptn/keptn/blob/master/cli/cmd/install.go#L131)

I will take care of getting the docu for the command fixed.

Comment (jetzlstorfer) [reply content/home/kubernetes-cluster-step2.md]: it is now changed

**Pull request merged**